packages/vaadin-virtualizer/src/debounce.js

```diff
@@ -0,0 +1,168 @@
+/**
```

Comment:
This file is a copy of https://github.com/Polymer/polymer/blob/master/lib/utils/debounce.js